src/main/java/com/car/app/boundary/RestApplication.java

```diff
@@ -28,7 +28,7 @@ public RestApplication() {
 	        OpenAPI oas = new OpenAPI();
 	        oas.setServers(null);
 	        Server server = new Server();
-	        server.setUrl("http://localhost:8080/carapp");
+	        server.setUrl("/carapp");
```

Comment:
Nice that you have removed the hard coded server address and port. Did you check the Application class ?